src/state/lists/hooks.ts

```diff
@@ -47,13 +48,16 @@ export function useAllLists(): AppState['lists']['byUrl'] {
 }
 
 function combineMaps(map1: TokenAddressMap, map2: TokenAddressMap): TokenAddressMap {
```

Comment:
this code is a little funny, we don't really need a key for each chain, just the chains present in either map1 or map2. would be a good candidate for adding unit tests